packages/studioCMS/src/integrations/studioCMSDashboard/routes/authroutes/login/signup.astro

```diff
@@ -26,44 +27,41 @@ if (locals.isLoggedIn) {
       </div>
     </center><br />
 
-      <!-- oAUTH Providers -->
       <OauthProviders dashboardConfig={dashboardConfig} />
-      <!-- End of oAUTH Providers -->
 
-      <!-- Username and Password Provider-->
         <div class="py-8">
           <div class="bg-base-300 p-8 rounded-lg md:w-[32rem]">
 
           <form method="post" action={registerAPI}>
 
-            <sl-input 
-            name="username" 
-            id="username">
-              <div class="text-xl font-bold" slot="label">Username</div>
-            </sl-input><br />
-
-            <sl-input 
-            name="email" 
-            id="email">
-              <div class="text-xl font-bold" slot="label">Email address</div>
-            </sl-input><br />
-
-            <sl-input 
-            name="displayname" 
-            id="displayname">
-              <div class="text-xl font-bold" slot="label">Display Name</div>
-            </sl-input><br />
-
-            <sl-input 
-            type="password" 
-            name="password"
-            id="password"
-            password-toggle>
-              <div class="text-xl font-bold" slot="label">Password</div>
-            </sl-input><br />
+            <Input 
+              name="username" 
+              id="username"
+              label='Username' 
+              /><br />
+
+            <Input 
+              name="email" 
+              id="email"
+              label='Email'
+              /><br />
+
+            <Input 
+              name="displayname" 
+              id="displayname"
+              label='Display Name' 
+              /><br />
+
+            <Input 
+              type="password" 
+              name="password"
+              id="password"
+              label="Password"
+              password-toggle 
+              /><br />
 
             <center>
```

Comment:
centah!

Reply:
This was not a CSS/Layout update, this was a Shoelace.style update